README.md

```diff
@@ -1,336 +1,79 @@
-out Lightning off-chain funds and you will receive Bitcoin on-chain funds in
-return. There will be some chain and routing fees associated with this swap.
-```
-NAME:
-   loop out - perform an off-chain to on-chain swap (looping out)
-
-USAGE:
-   loop out [command options] amt [addr]
-
-DESCRIPTION:
-
-  Attempts to loop out the target amount into either the backing lnd's
-  wallet, or a targeted address.
+## LND
+Note that Loop requires `lnd` to be built with **all of its subservers**. Download the latest [official release binary](https://github.com/lightningnetwork/lnd/releases/latest) or build `lnd` from source by following the [installation instructions](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md). If you choose to build `lnd` from source, use the following command to enable all the relevant subservers:
 
-  The amount is to be specified in satoshis.
-
-  Optionally a BASE58/bech32 encoded bitcoin destination address may be
-  specified. If not specified, a new wallet address will be generated.
-
-OPTIONS:
-   --channel value               the 8-byte compact channel ID of the channel to loop out (default: 0)
-   --addr value                  the optional address that the looped out funds should be sent to, if let blank the funds will go to lnd's wallet
-   --amt value                   the amount in satoshis to loop out (default: 0)
-   --conf_target value           the number of blocks from the swap initiation height that the on-chain HTLC should be swept within (default: 6)
-   --max_swap_routing_fee value  the max off-chain swap routing fee in satoshis, if not specified, a default max fee will be used (default: 0)
-   --fast                        Indicate you want to swap immediately, paying potentially a higher fee. If not set the swap server might choose to wait up to 30 minutes before publishing the swap HTLC on-chain, to save on its chain fees. Not setting this flag therefore might result in a lower swap fee.
 ```
-
-It's possible to receive more inbound capacity on a particular channel
-(`--channel`), and also have the `loop` daemon send the coins to a target
-address (`addr`). The latter option allows ones to effectively send on-chain
-from their existing channels!
-
-
-```
-loop out <amt_in_satoshis>
+make install tags="signrpc walletrpc chainrpc invoicesrpc"
 ```
 
-This will take some time, as it requires an on-chain confirmation. When the
-swap is initiated successfully, `loopd` will see the process through.
-
-To query in-flight swap statuses, run `loop monitor`.
+## Usage
 
-The loop client also has the ability to automatically dispatch loop out swaps on your behalf - see our [autoloop documentation](https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md) for details.
+### AutoLoop
+AutoLoop makes it easy to keep your channels balanced. Checkout our [autoloop documentation](https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md) for details.
```

Comment:
Could also link to gitbook if we want prettier docs: https://docs.lightning.engineering/advanced-best-practices/advanced-best-practices-overview/autoloop, it will only ever lag the by 24h and we can sync manually if we wanted to.

Reply:
SGTM!